### PR TITLE
add r-rstoolbox conda recipe

### PR DESCRIPTION
**Pull request by Marie59**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/r-rstoolbox/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    - r-terra
+
+test:
+  commands:
+    - $R -e "library('RStoolbox')"           # [not win]
+    - "\"%R%\" -e \"library('RStoolbox')\""  # [win]
+
+about:
+  home: https://bleutner.github.io/RStoolbox/, https://github.com/bleutner/RStoolbox
+  license: GPL-3
```
Comment (dbast) [inline recipes/r-rstoolbox/meta.yaml:81]:
```suggestion
  license: GPL-3.0-or-later
```

File: recipes/r-rstoolbox/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    - r-exactextractr
+    - r-foreach
+    - r-ggplot2
+    - r-lifecycle
+    - r-raster >=2.3_40
+    - r-reshape2
+    - r-rgdal
+    - r-sf
+    - r-sp
+    - r-terra
```
Comment (dbast) [inline recipes/r-rstoolbox/meta.yaml:52]:
```suggestion
    - r-terra
    - libblas
```

**Comment by Marie59** [comment]
@dbast thanks for your help ! Indeed windows is not required we can skip it

File: recipes/r-rstoolbox/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/RStoolbox_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/RStoolbox/RStoolbox_{{ version }}.tar.gz
+  sha256: e76bd8592e5230237deaacaebf50ba622ffa8b4556b46105a0dce725ea1a52f6
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
```
Comment (dbast) [inline recipes/r-rstoolbox/meta.yaml:17]:
```suggestion
  number: 0
  skip: True  # [win]
```

**Pull request merged**